library/src/nvcc_detail/hipsparse.cpp

```diff
@@ -3518,6 +3518,62 @@ hipsparseStatus_t hipsparseZcsric02(hipsparseHandle_t         handle,
                                                          pBuffer));
 }
 
+hipsparseStatus_t hipsparseSnnz(hipsparseHandle_t         handle,
+                                hipsparseDirection_t      dirA,
+                                int                       m,
+                                int                       n,
+                                const hipsparseMatDescr_t descrA,
+                                const float*              A,
+                                int                       lda,
+                                int*                      nnzPerRowColumn,
+                                int*                      nnzTotalDevHostPtr)
+{
+    return hipCUSPARSEStatusToHIPStatus(cusparseSnnz(
+        (cusparseHandle_t)handle, dirA, m, n, descrA, A, lda, nnzPerRowColumn, nnzTotalDevHostPtr));
```

Comment:
same here, should use a conversion function for `dirA`

Reply:
okay

Reply:
done.